app/models/miq_expression.rb

```diff
@@ -1101,7 +1100,7 @@ def self.value2human(val, options = {})
       friendly = tables.split(".").collect do|t|
         if t.downcase == "managed"
           val_is_a_tag = true
-          @company + " Tags"
+          "#{User.current_tenant.name} Tags"
```

Comment:
I didn't change anything here.

Currently [Tenant#name](https://github.com/ManageIQ/manageiq/blob/master/app/models/tenant.rb#L82) is defined as 
vmdb[:server][:company] for the root tenant.

So you would prefer `"#{Tenant.root_tenant.name} Tags"`?